README.md

```diff
@@ -1,36 +1,533 @@
 # xmtp-react-native-sdk
 
-Wrappers for native xmtp sdks for react native
+![Lint](https://github.com/xmtp/xmtp-android/actions/workflows/lint.yml/badge.svg) ![Status](https://img.shields.io/badge/Project_Status-Pre--Preview-red)
 
-## NOTES
+<!--eng: Correctly stated below? Necessary to say which native XMTP SDKs, or no?-->
 
-- To use the example app, you need to `npm install --force`
+This repo provides wrappers that help you build a React Native app with XMTP. The wrappers serve as a bridge between your React Native app and native XMTP SDKs.
```

Comment:
```suggestion
This repo provides a package you can use to build with XMTP in a React Native or Expo app.
```

I removed the mentions of the wrappers since that's just an implementation detail that devs shouldn't need to worry about.